### PR TITLE
Update API metrics to summary instead of guage

### PR DESCRIPTION
**Pull request by liamwazherealso**

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -94,7 +95,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Create a new Cmd to provide shared dependencies and start components
+	// Create a new Cmd to provide shared dependencies and start compone/home/lpieri/Dropboxnts
```
Comment (fahlmant) [inline cmd/manager/main.go]: Copy Paste error?

Comment (liamwazherealso) [reply cmd/manager/main.go]: Yes

File: pkg/localmetrics/localmetrics.go [hunk 2]
```diff
@@ -53,59 +56,57 @@ var (
...
-	// if there is an api key make an authenticated calld
-	if APIkey != "" {
+	// if there is an api key make an authenticated called
+	if APIKey != "" {
 		req, _ := http.NewRequest("GET", apiEndpoint, nil)
 		req.Header.Set("Accept", "application/vnd.pagerduty+json;version=2")
 		req.Header.Set("Content-Type", "application/json")
-		req.Header.Set("Authorization", "Token token="+APIkey)
-
+		req.Header.Set("Authorization", "Token token="+APIKey)
```
Comment (jharrington22) [inline pkg/localmetrics/localmetrics.go]: I prefer `fmt.Sprintf("Token token=%s", APIKey)` here as it feels more explicit and easier to read.

Comment (liamwazherealso) [reply pkg/localmetrics/localmetrics.go]: I'll take that suggestion

File: pkg/localmetrics/localmetrics.go [hunk 3]
```diff
@@ -31,19 +31,22 @@ const (
 
 var (
 	MetricPagerDutyCreateFailure = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "metric_pageduty_create_failure",
```
[the remark this answers is not in the document]
Comment (liamwazherealso) [reply pkg/localmetrics/localmetrics.go]: Agreed

**Comment by jharrington22** [comment]
cc/ @mwoodson @jewzaam CR please?

**Pull request merged**